src/Parser.js

```diff
@@ -527,7 +529,7 @@ export default class Parser {
         } else if (utils.contains(Parser.sizeFuncs, func)) {
             // If we see a sizing function, parse out the implicit body
             this.consumeSpaces();
-            const body = this.parseExpression(false);
+            const body = this.parseExpression(false, breakOnTokenText);
```

Comment:
Why does this only apply to this block and not to the rest below: `styleFuncs`, `oldFontFuncs`, and `\color`? Are they not similarly affected? Or do none of our functions currently want to allow these in optional groups?
Otherwise, this PR mostly LGTM.

Reply:
Good question. I was fixing only the issue, but I'm sure there are places where KA also uses `\color` inside of optional groups (apparently LaTeX supports this too).  LaTeX also supports style functions within optional blocks too.